migrator/src/main/java/com/redhat/lightblue/migrator/ThreadMonitor.java

```diff
@@ -229,7 +240,7 @@ public void run() {
                     runNow=false;
                 } else {
                     try {
-                        waiter.wait(threadTimeout);
+                        waiter.wait(60l*1000l);
```

Comment:
Done.


On Tue, May 3, 2016 at 10:38 AM, Brandon Vulaj <notifications@github.com>
wrote:

> In migrator/src/main/java/com/redhat/lightblue/migrator/ThreadMonitor.java
> <https://github.com/lightblue-platform/lightblue-migrator/pull/345#discussion_r61913385>
> :
>
> > @@ -229,7 +240,7 @@ public void run() {
> >                      runNow=false;
> >                  } else {
> >                      try {
> > -                        waiter.wait(threadTimeout);
> > +                        waiter.wait(60l*1000l);
>
> Put this in a var somewhere maybe.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/345/files/59e1f21dde5ef706406a8e0ae1d9ccfedaf8b15a#r61913385>
>